### PR TITLE
Warn against combined harms, but not development

### LINKED ISSUE #13: warn against combined harms, but not against parallel development and testing
> We are also wary of new approaches being introduced while third-party cookies are still available to authors, as this comes with a risk that more, parallel channels for privacy invasion become possible, rather than fewer. 

I think the intent here is to warn against adding new features which could allow additional cross-origin state that would likely be combined with third-party cookies, browser fingerprinting and other privacy-invasive means in ways that would be a long-term negative harm. I appreciate the emphasis that deprecation should happen (and personally preferred the earlier language about deprecation not being dependent on replacement).

Cautioning against the introduction, development or deployment of new approaches in parallel, though, seems to discourage discussion and testing, which have been very successful parts of web standards development and adoption. We want this transition to get going more quickly -- progress is overdue! -- and this could instead be interpreted as recommending waterfall-style development.

Maybe:

> We are also wary of new mechanisms being introduced that would be combined by some actors with cookies, fingerprinting surface or other tools in a way that would be abused for greater privacy invasion.

### PR DESCRIPTION
**Pull request by cynthia**
Resolves #13.

### CODE REVIEW

**Review by hober — APPROVED** [review]
👍

**Review by torgo — APPROVED** [review]
lgtm

File: index.html [hunk 1]
```diff
@@ -76,7 +76,7 @@
 
 Many varied proposals are being incubated in W3C Community Groups (eg. [PATCG](https://patcg.github.io/), [Privacy CG](https://github.com/privacycg/), [WICG](https://wicg.io/)) as well as outside (eg. [Privacy Sandbox](https://www.privacysandbox.com/)), and in these incubation stages multi-stakeholder support, consensus, and possible timelines for standardization are uncertain, and far from guaranteed.
 
-We are also wary of new approaches being introduced while third-party cookies are still available to authors, as this comes with a risk that more, parallel channels for privacy invasion become possible, rather than fewer.
+We are also wary of new mechanisms being introduced that would be combined by some actors with cookies, fingerprinting surface or other tools in a way that would be abused for greater privacy invasion.
```
Comment (hadleybeeman) [inline index.html]:
```suggestion
We are also wary of new mechanisms being introduced that could be abused together with cookies, fingerprinting surface or other tools, for greater privacy invasion.
```

**Review by torgo — APPROVED** [review]
lgtm

**Comment by npdoty** [comment]
🙏🏻

**Pull request merged**